### PR TITLE
fix: travis build

### PR DESCRIPTION
**Pull request by adityastic**
Fixed Travis

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -31,3 +31,5 @@ after_success:
   - bash scripts/prep-key.sh
   - 'if [ "$TRAVIS_PULL_REQUEST" == "false" ]; then bash scripts/upload-apk.sh; fi'
 
+before_install:
+  - yes | sdkmanager "platforms;android-28"
```
Comment (CloudyPadmal) [inline .travis.yml]: Instead try ${ANDROID_HOME}tools/bin/sdkmanager and see

**Review by CloudyPadmal — CHANGES_REQUESTED** [review]

File: .travis.yml (hunk 1 above)
Comment (adityastic) [reply .travis.yml]: the issue is with downloading the cache

**Comment by CloudyPadmal** [comment]
@adityastic beforeinstall and install executing the same thing here. Also sdkmanager is not known to system. So remove the beforeinstall part and remove / in install part as you've already done and update the PR. I think it'll build

**Comment by adityastic** [comment]
@CloudyPadmal the issue is with the cache, Travis fetches the caches from the main branch when building a pr. In this case for some reason it is not able to, My branch builds successfully, the issue will persist only when building PR. let me find something on this

**Comment by CloudyPadmal** [comment]
I see. Thanks @adityastic. I'll join in a while to fix it

**Comment by adityastic** [comment]
@CloudyPadmal Great, I'll try till then

**Comment by adityastic** [comment]
@CloudyPadmal can you check if the last build in this repo is cached or not

**Comment by adityastic** [comment]
@CloudyPadmal good to go

File: .travis.yml [hunk 2]
```diff
@@ -4,6 +4,8 @@ sudo: false
 
 jdk: oraclejdk8
 
+dist : trusty
```
Comment (jddeep) [inline .travis.yml:7]: @adityastic And we were trying so hard, different ways to fix the issue xD. wow.

**Review by jddeep — APPROVED** [review]

**Review by CloudyPadmal — APPROVED** [review]

**Comment by CloudyPadmal** [comment]
Nice. Thanks for the PR @adityastic

File: .travis.yml (hunk 2 above)
Comment (adityastic) [reply .travis.yml:7]: @jddeep haha yeah.

**Comment by adityastic** [comment]
@CloudyPadmal my pleasure

**Pull request merged**